src/metabase/api/activity.clj

```diff
@@ -189,11 +190,78 @@
 (def ^:private views-limit 8)
 (def ^:private card-runs-limit 8)
 
-#_{:clj-kondo/ignore [:deprecated-var]}
-(api/defendpoint-schema GET "/recent_views"
-  "Get the list of 5 things the current user has been viewing most recently."
+(defn- bookmarks
```

Comment:
maybe call this `bookmarks-query` or something? at first I expected it to actually return the bookmarks